test_rclcpp/test/test_services_client.cpp

```diff
@@ -21,8 +21,16 @@
 
 #include "test_rclcpp/srv/add_two_ints.hpp"
 
-TEST(test_services_client, test_add_noreqid) {
-  auto node = rclcpp::Node::make_shared("test_services_client");
+#ifdef RMW_IMPLEMENTATION
+# define CLASSNAME_(NAME, SUFFIX) NAME ## __ ## SUFFIX
+# define CLASSNAME(NAME, SUFFIX) CLASSNAME_(NAME, SUFFIX)
+#else
+# define CLASSNAME(NAME, SUFFIX) NAME
+#endif
```

Comment:
The CMake code doesn't set the compiler definition `RMW_IMPLEMENTATION` for this target yet.